lib/commands/v2new/ui.js

```diff
@@ -116,7 +116,12 @@ function getDeploymentType(deployType, callback) {
         pageSize: 30,
         filter: input => input.replace(/\n.*/g, '')
     }]).then((answer) => {
-        callback(null, answer.deployDelegate);
+        // eslint-disable-next-line array-callback-return
+        R.values(deployType).map((deployer) => {
+            if (answer.deployDelegate === deployer.OPTION_NAME) {
+                return callback(null, deployer.NAME);
+            }
+        });
```

Comment:
I think if not understanding that the loop will only hit the target once, putting the callback inside a for loop looks not good. I think you can use try
```
callback(null, R.find(R.propEq('OPTION_NAME', 'AWS Lambda'))(R.values(deployType)))
```

Reply:
This solution is much better, it returns the first element of the list which matches the predicate, or `undefined` if no element matches.